src/drivers/ms4525_airspeed/ms4525_airspeed.cpp

```diff
@@ -586,11 +560,11 @@ info()
 static void
 meas_airspeed_usage()
 {
-	warnx("usage: meas_airspeed command [options]");
-	warnx("options:");
-	warnx("\t-b --bus i2cbus (%d)", PX4_I2C_BUS_DEFAULT);
-	warnx("command:");
-	warnx("\tstart|stop|reset|test|info");
+	PX4_INFO("usage: meas_airspeed command [options]");
```

Comment:
@dagar Do you think it would be good to establish the new set of "good driver" habits to use the module macros here?

Reply:
Yes although I'm more interested in doing in centrally (launcher or interface) and then deleting all of this code :smile:.
Start with some notes for what you have in mind?